.github/workflows/iwyu.yml

```diff
@@ -35,11 +35,55 @@ jobs:
         run: |
           git config user.name orbitprofiler-bot
           git config user.email orbitprofiler-bot@google.com
+      - name: Check IWYU PR already open
+        id: check_has_open_iwyu_pr
+        uses: actions/github-script@v6
+        with:
+          github-token: '${{ secrets.ORBITPROFILER_BOT_PAT }}'
+          script: |
+            async function checkHasOpenIwyuPr() {
+              const {repo, owner} = context.repo;
+            
+              let page = 1;
+              let continue_requests = true;
+            
+              while (continue_requests) {
+                const result = await github.request(
```

Comment:
According to https://github.com/actions/github-script pagination is automatically handled by ``await github.paginate(github.rest.pulls.list, {owner, repo})``. This would simplify this part quite a bit.

Reply:
More details here: https://octokit.github.io/rest.js/v19#pagination

Reply:
That's great. That really simplified stuff a lot!